R/extract.R

```diff
@@ -9050,3 +9050,96 @@ extract.hurdle <- extract.zeroinfl
 #' @export
 setMethod("extract", signature = className("hurdle", "pscl"),
           definition = extract.hurdle)
+
+# -- extract.logitr (logitr) ---------------------------------------------------
+
+#' @noRd
+extract.logitr <- function(
```

Comment:
Can you format the function header in a way that is like the other functions? I know, it's nitpicking, but I'd like the formatting of the code to be consistent throughout the package, in line with what common style guides recommend. Thanks.